internal/ls/findallreferences.go

```diff
@@ -419,7 +419,7 @@ func (l *LanguageService) convertSymbolAndEntryToLocation(s *SymbolAndEntries) [
 	for _, ref := range s.references {
 		if ref.textRange == nil {
 			sourceFile := ast.GetSourceFileOfNode(ref.node)
-			ref.textRange = l.createLspRangeFromNode(ref.node, ast.GetSourceFileOfNode(ref.node))
```

Comment:
I can't tell if this is the right fix (`createLspRangeFromNode` -> `getRangeOfNode`) or if this is the right fix: https://github.com/microsoft/typescript-go/pull/1326/files#diff-4cd7fcdded6dbd561658f34fe1abb4030739555b30d90785a6ba6f08aec0d523R393